adamsjr8576/palette-picker-fe
# Project container setup

### What does this PR do?
- Adds the `ProjectCard` and `ProjectPalette` components 
- Adds the paletteCards
- Reformats the actions and reducers 

### How to test:
- pull down
- run `npm start`
- See the projects and palettes load 

### Issues:
- This PR closes no issues 
- The board has been updated 

### Notes:
- Per our discussion a few things will need to be tinkered with. The we need to create a new action/reducer or adding a palette.

Thanks!
@adamsjr8576
